v2/rscg_examples_site/docs/indexRSCG.md

```
@@ -20,6 +20,7 @@ This is the list of 291 ( 16 from Microsoft) RSCG with examples
 
 | No        | Name  | Date | Category |
 | --------- | ----- | ---- | -------- |  
+|292| [LayeredCraft.OptimizedEnums by LayeredCraft ](/docs/LayeredCraft.OptimizedEnums)|2026-09-06 => 06 September 2026 | [Enum](/docs/Categories/Enum) |
```

markdownlint-cli2 (0.23.2)

[warning] Spaces inside link text

(MD039, no-space-in-links)